### ISSUE TITLE
What coding languages/tools will we use?

### ISSUE THREAD

**Issue by Snaredrumhero** [Snaredrumhero/KillerSudoku#1]
What is the best language for our backend?
* C# is commonly used for web development and I'd like to use the opportunity to learn the language, but I can't see a reason why we'd want to use it over other languages since we're not going to be dealing with a lot of data

What is the best tool for creating a WebApp?
* Zach mentioned there was an extension in Visual Studio that will automatically load how the website will look as you code it, similar to how it works with Microsoft Apps

What is the best service for hosting an WebApp?
* I have an IT friend who rents space on Linux servers, I'll ask him for the services name

**Comment by ZacharyWolfe** [comment]
If we're looking for a C# backend we're probably going to have to use something like the ASP.NET framework on Visual Studio 2022 rather than Visual Studio Code, which sort of defeats the extension I mentioned here (https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer). I don't know the implications that using ASP.NET has on Electron if we are still considering that as well. Your IT friend sounds like a good outlet, if all else fails we can just run it locally and Chang should be okay with that. I also would enjoy learning more C#, just be open to also having to use JS as a back up route.

TLDR;
- VSCode will work if we go the JS, CSS route and are okay with not using C#, Electron should work well, and we have the live server extension.
- Visual Studio will be better suited for a C# backend, plus it will be more familiar to all of us and should still work with CSS etc, unknown Electron behavior.
